src/screens/Connectors/PaymentProcessor/ConnectPayPalFlow/PayPalFlowUtils.res

```diff
@@ -63,12 +63,7 @@ let handleObjectResponse = (~dict, ~setInitialValues, ~connector, ~handleStateTo
     ->getDictFromJsonObject
     ->getDictfromDict("connector_account_details")
     ->getString("auth_type", "")
-  let body = ConnectorUtils.generateInitialValuesDict(
-    ~values,
-    ~connector,
-    ~bodyType=bodyTypeValue,
-    ~isPayoutFlow=false,
-  )
+  let body = ConnectorUtils.generateInitialValuesDict(~values, ~connector, ~bodyType=bodyTypeValue)
```

Comment:
here you should pass the prop of connector type, introduce a new prop called connector type in handleObjectResponse method, as this is called from ConnectorHome and PayoutProcessorHome